var/spack/repos/builtin/packages/netcdf-c/package.py

```diff
@@ -123,6 +125,8 @@ class NetcdfC(AutotoolsPackage):
     # https://github.com/Unidata/netcdf-c/issues/250
     depends_on("hdf5@:1.8", when="@:4.4.0")
```

Comment:
This is just saying that if you use netcdf-c <= 4.4.0 you have to use hdf5 <= 1.8 because of the issue described here: https://github.com/Unidata/netcdf-c/issues/250

Reply:
Ah, ok thanks!